### PR TITLE
pkey secret

### PR DESCRIPTION
**Pull request by AjayTripathy**

### CODE REVIEW

File: cost-analyzer/values.yaml [hunk 1]
```diff
@@ -489,6 +489,7 @@ serviceAccount:
 #  serviceKeySecretName: "" # Use an existing AWS or Azure secret with format as in aws-service-key-secret.yaml or azure-service-key-secret.yaml. Leave blank if using createServiceKeySecret 
 #  createServiceKeySecret: true # Creates a secret representing your cloud service key based on data in values.yaml. If you are storing unencrypted values, add a secret manually
 #  sharedNamespaces: ""
-#  productKey: # apply business or enterprise product license
+  productKey: # apply business or enterprise product license. Supply as plaintext (this is never used to access your data) or as an existing secret
 #    key: ""
 #    enabled: false
+    secretname: productkeysecret # create a secret of format
```
Comment (dwbrown2) [inline cost-analyzer/values.yaml:495]: Is this an existing secret? If so, should we just call it `existingProductKeySecret`?

Also, I'm assuming we want this empty of commented out?

Comment (AjayTripathy) [reply cost-analyzer/values.yaml:495]: "existing" is a bit inconsistent with how we do this for serviceKeySecretName. Definitely needs to be commented by default though.

Comment (dwbrown2) [reply cost-analyzer/values.yaml:495]: I personally think how this is currently done is not intuitive.

**Comment by AjayTripathy** [comment]
Ajay Tripathy  1:55 PM
anyway, mind reviewing https://github.com/kubecost/cost-analyzer-helm-chart/pull/606/files to double check all that?

Matt Bolt  2:34 PM
Yeah, that -- that seems fine, but it's assuming a pre-created secret right?

Matt Bolt  2:39 PM
LGTM

**Pull request merged**